mypy/tvar_scope.py

```diff
@@ -120,7 +120,7 @@ def get_binding(self, item: Union[str, SymbolTableNode]) -> Optional[TypeVarLike
             return None
 
     def __str__(self) -> str:
-        me = ", ".join('{}: {}`{}'.format(k, v.name, v.id) for k, v in self.scope.items())
+        me = ", ".join(f'{k}: {v.name}`{v.id}' for k, v in self.scope.items())
         if self.parent is None:
             return me
-        return "{} <- {}".format(str(self.parent), me)
+        return f"{str(self.parent)} <- {me}"
```

Comment:
Is the call to `str()` needed here?

```suggestion
        return f"{self.parent} <- {me}"
```